recipes/capnproto/all/conanfile.py

```diff
@@ -0,0 +1,185 @@
+from conans import ConanFile, CMake, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import glob
+import os
+
+
+class CapnprotoConan(ConanFile):
+    name = "capnproto"
+    description = "Cap'n Proto serialization/RPC system."
+    license = "MIT"
+    topics = ("conan", "capnproto", "serialization", "rpc")
+    homepage = "https://capnproto.org"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ("CMakeLists.txt", "patches/**")
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_openssl": [True, False],
+        "with_zlib": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_openssl": True,
+        "with_zlib": True
+    }
+
+    _cmake = None
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "15",
+            "gcc": "5",
+            "clang": "5",
+            "apple-clang": "5.1",
```

Comment:
💭 5 seems like an odd number

Reply:
I'm not familiar with apple-clang, but It seems that XCode 5.1uses clang 3.4, which is the minimal version with full C++14 support. On the other hand, clang 3.4 is not sufficient for capnproto...

Reply:
Usually it's set to 9 got full c++14 but i am no expert either. Let's see what others reply